redirects.yaml

```diff
@@ -745,7 +745,7 @@ telcos/?: "/telecommunications"
 
 # Blog redirects
 blog/search/?: /search
-blog(?P<page>(/.+)?)/feed?: "https://admin.insights.ubuntu.com{page}/feed"
+# blog(?P<page>(/.+)?)/feed?: "https://admin.insights.ubuntu.com{page}/feed"
```

Comment:
remove line :smile:

Reply:
I left it a a reminder we need to deal with this pattern

Reply:
I removed it and created an issue in the blog repo